### PR TITLE
docs(tabs.mdx): documentation of the Tabs component

### PR DESCRIPTION
**Pull request by DominusKelvin**
## Description
Documentation of the Tabs component

### CODE REVIEW

**Comment by DominusKelvin** [comment]
@codebender828, please review.

File: packages/chakra-ui-docs/docs/tabs.mdx [hunk 1]
```diff
@@ -1 +1,334 @@
...
+<SEO
+  title="Tabs"
+  description="The Tab component consists of clickable tabs, that are aligned side by side."
+/>
+
+# Tabs
+
+An accessible tabs component.
+
+The `Tab` and `TabPanel` elements are associated by their order in the tree.
```
Comment (codebender828) [inline packages/chakra-ui-docs/docs/tabs.mdx:12]: Please change this to `CTab` and `CTabPanel`

File: packages/chakra-ui-docs/docs/tabs.mdx [hunk 2]
```diff
@@ -1 +1,334 @@
...
+
+## Props
+
+### Tabs Props
+
+Tabs composes `Box` so you call pass all `Box` related props.
+
+| Name           | Type                                                                              | Default      | Description                                                                                                                     |
+| -------------- | --------------------------------------------------------------------------------- | ------------ | ------------------------------------------------------------------------------------------------------------------------------- |
+| `onChange`     | `(index: number) => void`                                                         |              | The callback to update the active tab index.                                                                                    |
```
Comment (codebender828) [inline packages/chakra-ui-docs/docs/tabs.mdx:314]: This should be moved to the Events table. I think emits the @change event

File: packages/chakra-ui-docs/docs/tabs.mdx [hunk 3]
```diff
@@ -1 +1,334 @@
...
+
+## Import
+
+```js
+import { CTabs, CTabList, CTabPanels, CTab, CTabPanel } from '@chakra-ui/vue'
+```
+
+## Usage
+
+You can render any element within `Tabs`, but `TabList` should only have `Tab`
```
Comment (codebender828) [inline packages/chakra-ui-docs/docs/tabs.mdx:25]: Same here about prefixing component names with `C`

**Comment by DominusKelvin** [comment]
@codebender828 please review.

File: packages/chakra-ui-docs/docs/tabs.mdx (hunk 1 above)
Comment (codebender828) [reply packages/chakra-ui-docs/docs/tabs.mdx:12]: Didn't see this change.

File: packages/chakra-ui-docs/docs/tabs.mdx [hunk 4]
```diff
@@ -1 +1,339 @@
...
+| `variantColor` | `string`                                                                          |              | The primary color to use for the selected variant. Use a color key in `theme.colors`                                            |
+| `size`         | `sm`, `md`, `lg`                                                                  | `md`         | The visual size of the tabs                                                                                                     |
+| `orientation`  | `horizontal`, `vertical`                                                          | `horizontal` | The orientation of the tabs                                                                                                     |
+| `isFitted`     | `boolean`                                                                         |              | If `true`, the tabs will stretch to fill the available space                                                                    |
+
+### Tabs Events
+
+| Name       | Description              |
+| ---------- | ------------------------ |
+| `onChange` | Emits the @change event. |
```
Comment (codebender828) [inline packages/chakra-ui-docs/docs/tabs.mdx:327]: The event name is `@change`. Description is "Event emitted when the active tab changes"

File: packages/chakra-ui-docs/docs/tabs.mdx [hunk 5]
```diff
@@ -1 +1,339 @@
...
+        </c-tab-panel>
+        <c-tab-panel>
+            <p>two!</p>
+        </c-tab-panel>
+        <c-tab-panel>
+            <p>three!</p>
+        </c-tab-panel>
+        </c-tab-panels>
+    </c-tabs>
+</c-box>
```
Comment (codebender828) [inline packages/chakra-ui-docs/docs/tabs.mdx:51]: The code indentation here is different from how you normally write. So it's hard to read. Should be something like:

```vue
<c-box w="600px">
    <c-tabs>
      <c-tab-list>
        <c-tab>One</c-tab>
        <c-tab>Two</c-tab>
        <c-tab>Three</c-tab>
      </c-tab-list>

      <c-tab-panels>
        <c-tab-panel>
          <p>one!</p>
        </c-tab-panel>
        <c-tab-panel>
          <p>two!</p>
        </c-tab-panel>
        <c-tab-panel>
          <p>three!</p>
        </c-tab-panel>
      </c-tab-panels>
    </c-tabs>
  </c-box>
```

File: packages/chakra-ui-docs/docs/tabs.mdx (hunk 4 above)
Comment (DominusKelvin) [reply packages/chakra-ui-docs/docs/tabs.mdx:327]: Oh got it. My bad

File: packages/chakra-ui-docs/docs/tabs.mdx (hunk 5 above)
Comment (DominusKelvin) [reply packages/chakra-ui-docs/docs/tabs.mdx:51]: I often find issue leaving spaces in the code blocks

File: packages/chakra-ui-docs/docs/tabs.mdx [hunk 6]
```diff
@@ -1 +1,348 @@
...
+<SEO
+  title="Tabs"
+  description="The Tab component consists of clickable tabs, that are aligned side by side."
+/>
+
+# Tabs
+
+An accessible tabs component.
+
+The `Tab` and `TabPanel` elements are associated by their order in the tree.
```
Comment (codebender828) [inline packages/chakra-ui-docs/docs/tabs.mdx:12]: `CTab` and `CTabPanel`. Then this shall be good to go!

Comment (DominusKelvin) [reply packages/chakra-ui-docs/docs/tabs.mdx:12]: @codebender828 resolved!

**Pull request merged**